DEPS

```diff
@@ -267,7 +271,7 @@ allowed_hosts = [
 ]
 
 deps = {
-  'src': 'https://github.com/flutter/buildroot.git' + '@' + '8c274b21f1ad4f2aec0a5e0ae8f4264393045b4b',
+  'src': 'https://github.com/zanderso/buildroot.git' + '@' + '28e42a766db2ddc593aa9a61816f7af8aaecad5f',
```

Comment:
Do you want to wait for the buildroot PR to switch it over?

Reply:
Yes. After the buildroot PR lands, this can be switched over.

Reply:
Done.